src/main/java/com/liveperson/api/MessagingAgent.java

```diff
@@ -64,6 +72,9 @@
     @WebsocketSingleNotification("ms.MessagingEventNotification")
     WebsocketService.ListenerBuilder onNextMessagingEventNotification();
 
+    @WebsocketSingleNotification("cqm.SubscribeExConversationsResponse")
```

Comment:
``cqm.SubscribeExConversationsRespons`` is response, not notification. Please fix the annotation and the method name.

Reply:
Since it is a response you will get it anyway from the future of the request. No need to add it to the interface.